packages/server/utils/OpenAIServerManager.ts

```diff
@@ -21,20 +27,29 @@ class OpenAIServerManager {
     const textStr = Array.isArray(text) ? text.join('\n') : text
     try {
       const location = summaryLocation ?? 'retro meeting'
-      const response = await this.openAIApi.createCompletion({
-        model: 'text-davinci-003',
-        prompt: `Below is a comma-separated list of text from a ${location}. Summarize the text for a second-grade student in one or two sentences. When referring to people in the summary, do not assume their gender and default to using the pronouns "they" and "them".
+      const response = await this.openAIApi.createChatCompletion({
+        model: 'gpt-3.5-turbo',
+        messages: [
+          {
+            role: 'user',
+            content: `Below is a newline delimited text from a ${location}.
+            Summarize the text for a second-grade student in one or two sentences.
+            When referring to people in the summary, do not assume their gender and default to using the pronouns "they" and "them".
+            If you can't provide a summary, simply write the word "${OpenAIMagicWords.NO_SUMMARY_PROMPT_INSTRUCTION_WORDS}".
```

Comment:
-1 I've copied and pasted the reflections from two previous Retros. For one of them, it was perfect, for another, it returned "No" for a few of them, when it would normally attempt a summary. 

Below is an example of a group of reflections that it would always be able to create a summary for previously but now consistently returns "No".

![Screenshot 2023-07-24 at 15 58 34](https://github.com/ParabolInc/parabol/assets/39854876/6cfe907e-2584-46d4-a591-0259c9a86b99)


Perhaps it'd be best to remove this line?

Reply:
If we remove that command from the prompt, sometimes ChatGPT would return like "the text does not provide any information":
<img width="918" alt="Screenshot 2023-07-24 at 14 51 34" src="https://github.com/ParabolInc/parabol/assets/1879975/64eb5db2-7647-4a53-947f-162c2ce84d23">
I also got "I'm sorry, but the given text does not provide any information that can be summarized for a second-grade student" once.
So I think we need a deterministic null state response so on the customer side, we can provide the same apology.

I can also reproduce your example locally but when I try the exactly same prompt in ChatGPT GUI, I got consistent meaningful answers: 
<img width="590" alt="Screenshot 2023-07-24 at 14 54 22" src="https://github.com/ParabolInc/parabol/assets/1879975/0f12633f-4877-48f0-97df-45dc53ff5d16">
Does the GUI use GPT-4?
Because with the same set of parameters we have in the app, using `gpt-4` in the playground produces meaningful answer too:
<img width="855" alt="Screenshot 2023-07-24 at 14 57 22" src="https://github.com/ParabolInc/parabol/assets/1879975/f33ad36e-b392-4e95-ad6d-b16d0fb34aa3">

Reply:
> sometimes ChatGPT would return like "the text does not provide any information"

I feel like the summary that says, "This is just a bunch of random letters", is a great summary. I often enjoy seeing what it comes up with when there are random reflections like this. 

> "I'm sorry, but the given text does not provide any information that can be summarized for a second-grade student" once.

I think that's a bad summary as it mentions "second-grade student" which is confusing. Maybe we could remove "second-grade student" from the prompt 🤔 

I lean towards not including the `simply write the word "No"` bit of the prompt to encourage more summaries, but feel free to include it if you feel strongly. 

> Does the GUI use GPT-4?

I think you can specify whether ChatGPT should use 3.5 or 4. Maybe the other settings are throwing it off?